mailbox/spamassassin/src/main/java/org/apache/james/mailbox/spamassassin/SpamAssassinListener.java

```diff
@@ -69,7 +68,7 @@ public void event(Event event) {
 
     @VisibleForTesting
     boolean isEventOnSpamMailbox(Event event) {
-        if (event.getMailboxPath().getName().equalsIgnoreCase(SPAM_MAILBOX_NAME)) {
+        if (event.getMailboxPath().getName().equalsIgnoreCase(DefaultMailboxes.SPAM)) {
```

Comment:
It's not what I asked for. I want a class that can verify what the role of the mailboxpath is.
Because when we'll rely on mailbox annotation, this code will break (and even worse, no test will catch it)